bundles/org.openhab.binding.konnected/README.md

```diff
@@ -16,13 +16,21 @@ The binding will then create things for each module discovered which can be adde
 
 ## Thing Configuration
 
-The binding attempts to discover The Konnected Alarm Panels via the UPnP service.
-The auto-discovery service of the binding will detect the ip address and port of the Konnected Alarm Panel.
+The binding attempts to discover The Konnected Alarm Panels via the UPnP service. 
+The auto-discovery service of the binding will detect the base URL of the Konnected Alarm Panel. 
+When manually adding things the base URL of the Konnected Alarm Panel will need to be configured. 
+The base URL should include scheme, address and port (for example http://192.168.1.123:9123).
+
 The binding will attempt to obtain the ip address of your openHAB server as configured in the OSGi framework.
-However, if it is unable to determine the ip address it will also attempt to use the network address service to obtain the ip address and port.
+If it is unable to determine the ip address it will also attempt to use the network address service to obtain the ip address and port.
```

Comment:
```suggestion
If it is unable to determine the IP address it will also attempt to use the network address service to obtain the IP address and port.
```